modules/UI.js

```diff
@@ -1,6 +1,6 @@
-// eslint-disable-next-line import/prefer-default-export
-export class UI {
-// eslint-disable-next-line class-methods-use-this
+/* eslint-disable class-methods-use-this */
```

Comment:
- [ ] All so this part, I would strongly recommend you fix it up and remove the disabling lines as this will contribute to your professionalism and gain some good ways of writing a clean code in the near future, what do you think about it? 😃